exploration_map_manager:
  ros__parameters:
    map_resolution: 1.0
    map_width: 50
    map_height: 50
    perception_fov: 0.95
    perception_distance: 4.0